src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeDefinition.xml

```diff
@@ -85,6 +85,7 @@
       <Property Name="IsExternal" ValidFor="Sql2016|Sql2017|AzureV12"/>
     </Properties>
     <Child Name="SystemTables" IsSystemObject="1"/>
+    <Child Name="ExternalTables"/>
   </Node>
```

Comment:
I agree with Kevin - I'd prefer to keep existing behavior where external tables are just normal tables under the tables folder with an extra (External) tag added